### DISCUSSION TITLE
auth/security thing

### DISCUSSION THREAD

**Discussion by atherdon** [atherdon/dashb#55] [Ideas]
SO, while it's not important right now, I think at some point in the future - we need to have some sort "secure version".

and yeah - I don't want to reinvent the wheel with login forms.
we'll have like 10 users max, and for those reasons - thing like auth0 will be free

https://auth0.com/blog/complete-guide-to-react-user-authentication/

**Comment by atherdon** [comment]
@master-bogdan do you thinks we can't do t without database, right?

**Reply by master-bogdan** [reply]
@atherdon we can, but in future this can lead to a huge changes, when we will be need to integrate database in app

**Reply by atherdon** [reply]
at this point i dont worried about problems that we might face in March/April. Because we don't know how this project will be accepted by my teammates. So my goal is to ship as much stuff as possible before March.
Right now it's Jan 16th and we already behind my schedule. So the more "views" even without complex logic we can ship - the better is for everyone. 
later probably we'll re-arrange a lot of stuff inside. But let's deliver fancy pages first.

**Comment by master-bogdan** [comment]
@atherdon ok, I will make simple auth for start

**Reply by atherdon** [reply]
@master-bogdan  let's explore if auth0 will work. I hate login forms and lazy to handle all of that "stuff". If Auth0 isn't a good option for us right now(meaning that it can't work with some simple fileDB) - let's leave it for the future.
again - everything that is simple - will be our first priority. everything else - need to be discussed

**Comment by master-bogdan** [comment]
@atherdon
If we need real simple Auth, I can write simple nodejs API for this with express.
auth0 is more serious think, anyway I can integrate auth0, but I think it must be registration not from my account

**Reply by atherdon** [reply]
i dont want those complex things - like nodejs api, etc, etc. It's a way for normal people ;)

lets pause it for now. I'll try to think about it more

**Comment by vadim9999** [comment]
@master-bogdan How long have you been working with auth0? Is it a commercial experience?

**Reply by master-bogdan** [reply]
It will be new for me experience

**Comment by atherdon** [comment]
credentials: https://github.com/atherdon/dashb/blob/main/docs/readme.md
secret is hidden under comments(open in raw view)

**Comment by master-bogdan** [comment]
@vadim9999 is exist some test login info for auth ?

**Reply by master-bogdan** [reply]
@vadim9999 I remember you about test login, because all work now stopted. I don't want spent time to comment all auth thinks to get possibility for work on project

**Reply by vadim9999** [reply]
@master-bogdan I don't have test account for now. Comment all auth code that stops

**Reply by atherdon** [reply]
lets move out oatuh0 think into separated branch, so it wouldn't break our current speed andp processes

**Reply by vadim9999** [reply]
@master-bogdan I have just commented all authentication code
